### ISSUE TITLE
PPC64BE support

### ISSUE THREAD

**Issue by opilar** [conan-io/conan#668]
Started using conan on ppc64 machine:
```
-sh-4.2$ conan test_package
ERROR: 'ppc64' is not a valid 'settings.arch' value.
Possible values are ['armv6', 'armv7', 'armv7hf', 'armv8', 'ppc64le', 'x86', 'x86_64']
Check your settings ~/.conan/settings.yml or read the docs FAQ
-sh-4.2$ uname -a
Linux dev-ppc-build.unix.tensor.ru 3.10.0-327.22.2.el7.ppc64 #1 SMP Thu Jun 9 10:31:21 EDT 2016 ppc64 ppc64 ppc64 GNU/Linux
```
Seems like `ppc64` architecture needed.

**Comment by lasote** [comment]
Will be available in 0.16, in the meantime you can add it to your ~/.conan/settings.yml

**Comment by memsharded** [comment]
Please not that ppc64le was already there, I guess you want to have the big-endiand ppc64 architecture.

As a brief explanation, the ``settings.yml`` file is just to pre-define some common settings, but it can be modified to support any custom configuration of settings, and as long as they are shared with the team or collaborators, it will work.

**Comment by opilar** [comment]
Thank you.
I've found a bug on windows. I will contribute tomorrow.

**Comment by memsharded** [comment]
Hi! Lets keep the issue open, we label them as fixed if implemented, but don't close them until they are released. This one will be released in next 0.16, thanks!

**Comment by memsharded** [comment]
Released in 0.16
